modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -1,2146 +0,0 @@
-/* global __filename, RTCIceGatherer, RTCIceTransport, RTCDtlsTransport,
-RTCRtpSender, RTCRtpReceiver */
-
-import { getLogger } from 'jitsi-meet-logger';
-import yaeti from 'yaeti';
```

Comment:
you can also remove this dependency from package.json

Reply:
Removed it.